framework-plugins/lisk-framework-dashboard-plugin/src/ui/app.tsx

```diff
@@ -1,10 +1,12 @@
 import * as React from 'react';
 import { Provider } from 'react-redux';
 import { store } from './store';
-import { Main } from './pages/main';
+import MainPage from './pages/MainPage';
 
```

Comment:
this file needs to be renamed too?

Reply:
It seems to be renamed. 
![image](https://user-images.githubusercontent.com/112468/112017843-4b1ed380-8b2e-11eb-9407-db6756cdfeb5.png)

Reply:
ah sorry wrong line. `App.tsx` and `Text/index.tsx` is not changed